metals/src/main/scala/scala/meta/internal/metals/Trace.scala

```diff
@@ -0,0 +1,107 @@
+package scala.meta.internal.metals
+
+import java.io.PrintWriter
+import java.nio.file.Files
+import java.nio.file.StandardOpenOption
+
+import scala.util.Failure
+import scala.util.Success
+import scala.util.Try
+
+import scala.meta.internal.io.PathIO
+import scala.meta.io.AbsolutePath
+
+import dev.dirs.ProjectDirectories
+
+/**
+ * Manages JSON-RPC tracing of incoming/outgoing messages via BSP and LSP.
+ */
+object Trace {
```

Comment:
Could you rename the file using git (I think https://git-scm.com/docs/git-mv should work) so that it's possible to see what is the actual change.